recipes/ggd/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: ggd
+  version: {{ version }}
+
+source:
+  url: https://github.com/gogetdata/ggd-cli/archive/v{{ version }}.tar.gz
+  sha256: 540c67b4beb9dee34fa07c55e86e6f58c17dbc345a4700207cbc87615040a7ff
+
+build:
+    number: 0
+    noarch: python
+    entry_points:
+      - ggd = ggd.__main__:main
+    script: "python -m pip install . --no-deps --ignore-installed -vv" 
+
+requirements:
+  host:
+    - pip
+    - pyyaml
+  run:
+    - pip
```

```suggestion
    - python
```
Unless `ggd` uses `pip` directly (a GitHub search on your repo didn't yield anything), we can remove this direct dependency. (If `conda` uses `pip`, that's fine since it's a `conda` dependency.)
We should have `python` here because we have a `python` package ;).

---

It does not use pip directly

---

Good, then we can omit that dependency here :)